src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -568,6 +595,34 @@ TEST_F(ResumeCtrlTest, SetupDefaultHMILevel) {
   res_ctrl->SetupDefaultHMILevel(app_mock);
 }
 
+TEST_F(ResumeCtrlTest, ApplicationResumptiOnTimer_AppInFull) {
+  EXPECT_CALL(app_mngr_, application(_))
```

Comment:
@LitvinenkoIra use ON_CALL. Pleasee look throw tests and use on_call for not critical getters. But specify value with what function will be called.